src/News/TwitterWatcher.php

```diff
@@ -33,35 +34,35 @@ public function onArticlePublished(NodeProjectedEvent $pbjxEvent): void
         $this->notifyTwitter($event, $node, $pbjx);
     }
 
-    protected function createTwitterNotification(Message $event, Message $article, Pbjx $pbjx): Message
+    protected function createTwitterNotification(Message $event, Message $article, Message $app, Pbjx $pbjx): Message
     {
         $date = $event->get('occurred_at')->toDateTime()->add(new \DateInterval('PT180S'));
         $contentRef = $article->generateNodeRef();
+        $appId = $app->generateNodeRef()->getId();
         $id = UuidIdentifier::fromString(
             Uuid::uuid5(
                 Uuid::uuid5(Uuid::NIL, 'twitter-auto-post'),
-                $contentRef->toString()
+                $contentRef->toString().$appId
             )->toString()
         );
 
         return MessageResolver::resolveCurie('*:notify:node:twitter-notification:v1')::create()
             ->set('_id', $id)
-            ->set('title', $article->get('title'))
+            ->set('title', $article->get('title').'-'.$appId)
```

Comment:
@meghedi lets do a pipe delimiter here instead of a dash: `' | '`. might do title too if Greg is cool with it

Reply:
@efjacobson @meghedi i'd rather see title... like `some title of thingy (@handle)`  something nice and human friendly

Reply:
@efjacobson yeah thats what i meant about using the title but the title is not guaranteed to be unique. easy convention to adhere to and i think its unlikely that anyone would dupe titles on prod. seems easier to use that than the actual handle which i dont think is stored anywhere

Reply:
@efjacobson @gdbrown can you please see my changes, is that how you wanted.

Reply:
@gdbrown thoughts?